### PR TITLE
Add first simple unittests for english and danish

### PR DESCRIPTION
**Pull request by tgc-dk**
## Summary by CodeRabbit

- **New Features**
	- Enhanced the accuracy of Danish number pronunciation, including corrections to specific phrases and improved logic for number representation.
- **Tests**
	- Added comprehensive unit tests for Danish and English number parsing and pronunciation functionalities, covering various inputs and edge cases.
	- Introduced a new GitHub Actions workflow for automated unit testing across multiple Python versions.

### CODE REVIEW

File: tests/test_number_parser_en.py [hunk 1]
```diff
@@ -0,0 +1,25 @@
+import unittest
+from ovos_number_parser.numbers_en import numbers_to_digits_en, pronounce_number_en, pronounce_number_en
```
[the remark this answers is not in the document]
Comment (tgc-dk) [reply tests/test_number_parser_en.py]: fixed

**Comment by JarbasAl** [comment]
awesome work!

can you also add a .yml to .github/workflows so these tests run on every PR?

eg. something like this one https://github.com/OpenVoiceOS/ovos-utils/blob/dev/.github/workflows/unit_tests.yml

**Comment by JarbasAl** [comment]
by the way, this repository was split from lingua-franca and we have plenty more tests to port over to this repo https://github.com/OpenVoiceOS/ovos-lingua-franca/tree/dev/test/unittests 

no danish support there but might be useful

**Comment by tgc-dk** [comment]
I've added the unittest workflow, should be a good starting point :)
